vllm/engine/multiprocessing/client.py

```diff
@@ -616,11 +616,11 @@ async def _process_request(
                 build_guided_decoding_logits_processor_async(
```

Comment:
Should this method be renamed to structured output too?

Reply:
This is not user facing, and it is in v0, so probably not needed to rename it here now (given that the code path for v0 still calls it guided_decoding)